extensions/github/src/components/DiscussionListItem.tsx

```diff
@@ -0,0 +1,68 @@
+import { List, Icon, Color, ActionPanel, Action, Image } from "@raycast/api";
+
+import { DiscussionFieldsFragment } from "../generated/graphql";
+
+function getDiscussionIcon(discussion: DiscussionFieldsFragment):
+  | {
+      value: Image.ImageLike | undefined | null;
+      tooltip: string;
+    }
+  | Image.ImageLike
+  | undefined {
+  const categoryText = discussion.category?.name ? `Category: ${discussion.category?.name}` : "";
```

Comment:
Maybe instead of returning an empty string, it could be `"Unknown` instead? What do you think?